src/app/components/dashboard/FormikField/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { ErrorMessage, Field } from "formik";
+// @ts-ignore
```

Comment:
We shouldn't have ts-ignore here. Instead, we should update the tsconfig.json file.